doc.go

```diff
@@ -599,6 +599,29 @@ Note: As of February 2020, Golang's official library does not support passcode-e
 For security purposes, Snowflake highly recommends that you store the passcode-encrypted private key on the disk and
 decrypt the key in your application using a library you trust.
 
+# External browser authentication
+
+The driver allows to authenticate using the external browser.
+
+When a connection is created, the driver will open the browser window and ask the user to sign in.
+
+To enable this feature, construct the DSN with field "authenticator=EXTERNALBROWSER" or using a Config structure with
+following Authenticator specified:
+
+	config := &Config{
+		...
+		Authenticator: AuthTypeExternalBrowser,
+	}
+
+The external browser authentication implements timeout mechanism. This prevents the driver to hang interminably when
```

Comment:
prevents to? Or rather prevents from hanging?

Reply:
True, changed.